src/routes/books/utils.js

```diff
@@ -0,0 +1,56 @@
+const { check } = require("express-validator");
+
+const getErrorMessage = (id = null, title=null, status=404) => {
+    let error = "The book doesn't exist"
+    if (id || id === 0) {
+        error = `The book with the id: ${id} doesn't exist`
+    }
+    if (status === 404 && title) {
+        error = `The book with the title: ${title} doesn't exist`
+    }
+    if (status === 409 && title) {
+        error = `The book with the title: ${title} already exist`
+    }
+    return error;
+}
```

Comment:
This is more of a stylistic feedback:

While I see why you would want to extrapolate this into `util` to avoid repetition, this seems too specific to belong in`util`. I would put this closer to where the errors are handled (in this case, the controller); or, if the error messages are going to be dependent on status code, putting it where the status codes are defined will be appropriate too.